main/consolekit2/APKBUILD

```diff
@@ -1,24 +1,31 @@
-# Contributor: Bart Ribbers <bart.ribbers@openmailbox.org>
-# Maintainer: Bart Ribbers <bart.ribbers@openmailbox.org>
+# Contributor: Bart Ribbers <bribbers@disroot.org>
+# Maintainer: Bart Ribbers <bribbers@disroot.org>
 pkgbase=ConsoleKit2
 pkgname=consolekit2
 pkgver=1.2.0
-pkgrel=1
-pkgdesc="A framework for defining and tracking users, login sessions, and seats. "
+pkgrel=2
+pkgdesc="A framework for defining and tracking users, login sessions, and seats"
+provides=consolekit
```

Comment:
We might want to do `provides="consolekit=$pkgver"` so this gets automatically installed to satisfy `consolekit` dependency.